### PR TITLE
Legacy user

### PR DESCRIPTION
**Pull request by bcvandendool**
needs some slight refactoring and better tests
To run add another database called cosmos_website_test_legacy with user cosmos_website_tester_legacy and pw 2020123 in the same way as the main database. Then run `python manage.py migrate --database legacy` to migrate the legacy database.

### CODE REVIEW

File: cosmos/urls.py [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+    # Authentication Views
+    # https://docs.djangoproject.com/en/3.0/topics/auth/default/#using-the-views
+    # TODO consider manualy adding URL's to only allow password_reset or _change
+    path("accounts/", include("django.contrib.auth.urls")),
+    # --- Sitemap --- #
     url(r"^sitemap\.xml$", sitemap, {"sitemaps": {"cmspages": CMSSitemap}}),
 ]
 
-urlpatterns += i18n_patterns(url(r"^admin/", admin.site.urls), url(r"^", include("cms.urls")),)  # NOQA
+urlpatterns += i18n_patterns(url(r"^admin/", admin.site.urls), url(r"^", include("cms.urls")),)
```
Comment (dasyad00) [inline cosmos/urls.py:36]: since you disabled `i18n`, do we need `i18n_patterns`?

File: cosmos/test/test_legacy.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from django.utils import timezone
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from legacy.models import AuthUser, MysiteProfile
+from legacy.tokens import account_import_token
+
+
+class LegacyTestCase(TestCase):
+    databases = {"default", "legacy"}
```
Comment (dasyad00) [inline cosmos/test/test_legacy.py:12]: Forgot to add this in my review, but pls add the following tests:

- user sucessfully migrated
- user already migrated
- user migrating with different emails
  - success when - `@student.tue.nl`, `@tue.nl`
  - fail when - `@cosmostue.nl`, `@gmail.com`

**Comment by maximekabel** [comment]
@bcvandendool is there a way where we can actually test if this works?

**Comment by bcvandendool** [comment]
@maximekabel Ive added instruction on how to setup the other database to the main comment of the pr, will add documentation to the wiki later with explicit details on how to set it up.

**Pull request merged**